lib/queue.js

```diff
@@ -45,8 +45,13 @@ class Queue extends Emitter {
       });
     }
 
-    this._delayedTimer = new EagerTimer(this.settings.nearTermWindow);
-    this._delayedTimer.on('trigger', this._activateDelayed.bind(this));
+    // To avoid changing the hidden class of the Queue.
+    this._delayedTimer = this.settings.activateDelayedJobs
```

Comment:
Yea, I'm also doubtful it would have any performance benefit since it's just 1 vs 2, not 1 vs 20 or log(n) or n - but the "always see all properties in REPL" bit is a fair point that I hadn't considered. Sounds good as is.